CIScripts/Build/BuildFunctions.ps1

```diff
@@ -137,8 +137,27 @@ function Invoke-DockerDriverBuild {
     })
 
     $Job.Step("Intalling MSI builder", {
+        # go-msi needs specific version of its dependencies,
```

Comment:
We think it would be better to use MSIs to install go-msi, dep and ginkgo on builder in ansible scripts.

Reply:
For ginkgo you can:
1) Prebuild it and install it.
2) Do nothing.

Reply:
Moved this change to https://github.com/codilime/contrail-windows-ci/pull/13

I'm going to remove this commit and rebase.